src/models/cats.js

```diff
@@ -0,0 +1,13 @@
+const { Sequelize } = require("sequelize");
+
+module.exports = (sequelize, DataTypes) => {
+    const schema = {
+        name: DataTypes.STRING,
+        breed: DataTypes.STRING,
+        markings: DataTypes.STRING,
+        lastFed: DataTypes.DATE
+    }
+
+    return sequelize.define('Cat', schema)
+
+}
```

Comment:
Minor formatting change here, there shouldnt be any blank space after the `return` statement as it is the end of the function
```
    }

    return sequelize.define('Cat', schema)
}
```